mytardis_rocrate_builder/rocrate_dataclasses/rocrate_dataclasses.py

```diff
@@ -1,11 +1,18 @@
 """Definition of RO-Crate dataclasses"""
 
+import uuid
 from abc import ABC
 from dataclasses import dataclass
 from datetime import datetime
 from enum import Enum
 from pathlib import Path
-from typing import Any, Dict, List, Optional
+from typing import Annotated, Any, Dict, List, Literal, Optional, Union
+
+from pydantic import AfterValidator, Field, PlainSerializer, WithJsonSchema
+from slugify import slugify
+from validators import url
+
+from .. import MYTARDIS_NAMESPACE_UUID
```

Comment:
My preference is to use absolute references, though I know relative ones can be used. I'm guessing there are no "structural" changes in the works so this is not likely to cause an issue :)